mysql-test/suite/tokudb/t/change_column_char.test

```diff
@@ -1,4 +1,3 @@
-source include/have_tokudb.inc;
```

Comment:
Good catch, this is a generated test and the generators were never changed to insert the have_tokudb.inc when we added it.